cmd_unix.go

```diff
@@ -0,0 +1,39 @@
+// +build darwin dragonfly freebsd linux nacl netbsd openbsd solaris
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"syscall"
+)
+
+// Default location of hosts-file on Unix.
+var hostsFile = "/etc/hosts"
```

Comment:
Can we make this a package level constant on `lib` similar to `os.FileSeparator`?

Reply:
Yes. It'll have to be a variable on Windows, though...

Reply:
Er, right, a variable, not a constant, e.g.

```go
// +build darwin,...

var Location = "/etc/hosts"
```

```go
// +build windows

var Location = filepath.Join(os.Getenv(...), ...)
```